src/Executables/Examples/RandomAmr/RandomAmr.hpp

```diff
@@ -73,20 +96,44 @@ struct RandomAmrMetavars {
                              amr::Initialization::Domain<volume_dim>,
                              amr::Initialization::Initialize<volume_dim>>,
                          Parallel::Actions::TerminatePhase>>,
+          Parallel::PhaseActions<Parallel::Phase::CheckDomain,
+                                 tmpl::list<::amr::Actions::SendAmrDiagnostics,
+                                            Parallel::Actions::TerminatePhase>>,
+
           Parallel::PhaseActions<
-              Parallel::Phase::CheckDomain,
-              tmpl::list<::amr::Actions::SendAmrDiagnostics,
-                         Parallel::Actions::TerminatePhase>>>>;
+              Parallel::Phase::Evolve,
+              tmpl::list<PhaseControl::Actions::ExecutePhaseChange>>>>;
 
   using component_list =
-      tmpl::list<amr::Component<RandomAmrMetavars>, amr_element_component>;
+      tmpl::list<amr::Component<RandomAmrMetavars>, dg_element_array>;
 
   // NOLINTNEXTLINE(google-runtime-references)
   void pup(PUP::er& /*p*/) {}
 };
 
+template <typename Component>
+void register_amr_callbacks() {
```

Comment:
Do you have plans to move this to a reusable place?

Reply:
Yes, as I don't want to copy it into all the executables...  I'll do this in my next AMR PR as well